braindecode/preprocessing/mne_preprocess.py

```diff
@@ -0,0 +1,71 @@
+"""Preprocessor objects based on mne methods."""
+# Authors: Bruna Lopes <brunajaflopes@gmail.com>
+#        : Bruno Aristimunha <b.aristimunha@gmail.com>
+#
+# License: BSD-3
+import inspect
+import mne.io
+from braindecode.preprocessing import Preprocessor
+from braindecode.util import _update_moabb_docstring
+
+
+def generate_init_method(func):
```

Comment:
maybe we can make this function and `generate_mne_pre_processor` private?

Reply:
Yes!